source/app/dumping.cpp

```diff
@@ -672,6 +743,42 @@ void dumpOnlineFiles() {
     else showDialogPrompt(L"Failed to dump the online files...", L"OK");
 }
 
+void dumpSpotpass() {
+    std::vector<std::shared_ptr<titleEntry>> queue;
+    dumpingConfig spotpassConfig = { .dumpTypes = (DUMP_TYPE_FLAGS::CUSTOM) };
+
+    if (!showOptionMenu(spotpassConfig, false)) return;
+
+    titleEntry slcSpotpassDir{ .shortTitle = L"SpotPass Directory", .customFolder = folderPart{.inputPath = convertToPosixPath("/vol/storage_mlc01/usr/save/system/boss"), .outputPath = "/SpotPass Files"} };
+    queue.emplace_back(std::make_shared<titleEntry>(slcSpotpassDir));
+
+    dumpFileFilter filter{ .fileNames = {"task.db"} };
+    if (dumpQueue(queue, spotpassConfig, &filter)) showDialogPrompt(L"Successfully dumped all of the SpotPass files!", L"OK");
+    else showDialogPrompt(L"Failed to dump the SpotPass files...", L"OK");
+
+    wchar_t promptMessage[256];
+    swprintf(promptMessage, 256, L"Do you want to anonymously upload the SpotPass files to an archival server?\nIt will upload %d files of 1MB each!\n", filter.outMatchedFiles.size());
```

Comment:
The phrasing here seems a little vague. Could you possibly change it to say something like "Do you want to anonymously upload the files to the SpotPass Archival Project for data preservation?"

Reply:
The screen doesn't have infinite width, can you provide another text?